frontend/src/components/pages/Dashboard/index.tsx

```diff
@@ -22,13 +24,17 @@ const Dashboard = (): JSX.Element => {
   };
 
   React.useEffect(() => {
+    if (!authenticatedUser) {
+      return;
+    }
```

Comment:
```suggestion
    if (!authenticatedUser) {
      return <Redirect to={LOGIN_PAGE}/>;
    }
```

Reply:
this is inside useEffect function, we cannot return `<Redirect/>`